completion/tests/test_models.py

```diff
@@ -142,3 +142,69 @@ def test_submit_batch_completion_without_waffle(self):
         with self.assertRaises(RuntimeError):
             blocks = [(self.block_key, 1.0)]
             models.BlockCompletion.objects.submit_batch_completion(self.user, self.course_key, blocks)
+
+
+class BatchCompletionMethodTests(CompletionSetUpMixin, TestCase):
```

Comment:
This class could be named better - it would be easy to confuse it for tests corresponding to the `submit_batch_completion()` method, rather than the class methods it actually tests.  `CompletionFetchingTestCase` maybe, or something else you can think of?

Reply:
Sure!